configs/next/packages/gcc/stage_1

```diff
@@ -34,7 +34,6 @@ atcfg_pre_hacks() {
 		cp ${ATSRC_PACKAGE_WORK}/gcc/REVISION ${ATSRC_PACKAGE_WORK}/gcc/REVISION.back
 		sed -i "s/ \([^ ][^ ]*\)]$/ ${ATSRC_PACKAGE_REV}, based upon FSF revision \1]/" \
 			${ATSRC_PACKAGE_WORK}/gcc/REVISION
-		cmp --silent ${ATSRC_PACKAGE_WORK}/gcc/REVISION.back ${ATSRC_PACKAGE_WORK}/gcc/REVISION || exit 1
```

Comment:
The master branch of GCC does not have this file (gcc/REVISION), so I'm wondering what problem is this hunk trying to fix.  Is there a problem, at all?

Maybe the whole if-else block should be replaced with:

```
if [ ! -s ${ATSRC_PACKAGE_WORK}/gcc/REVISION]; then
  echo "[revision ${ATSRC_PACKAGE_REV}]" > ${ATSRC_PACKAGE_WORK}/gcc/REVISION
fi
```
(notice the inclusion of the `!` in the if statement)